api/app/models.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
```

Comment:
Do you think naming this file request_models or schemas might be a little clearer @aryanbhosale?

Reply:
Yeah even I feel so , schemas sounds better